fixtures/hive/bee-50-c.json

```
@@ -0,0 +1,547 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Bee 50-C",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2021-02-09",
+    "lastModifyDate": "2021-02-09"
+  },
+  "links": {
+    "manual": [
+      "https://drive.google.com/file/d/1pJK6BaAhvOGTwMP1VhfO8GH25YicCDwS/view"
+    ],
+    "productPage": [
+      "https://hivelighting.com/c-series/bee-50-c/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=HHbsbvlReC0"
+    ]
+  },
+  "physical": {
+    "dimensions": [100, 100, 136],
+    "weight": 0.7,
+    "power": 40,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [13, 180]
+    }
+  },
+  "availableChannels": {
+    "Intensity": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "CCT": {
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "1650K",
+        "colorTemperatureEnd": "8025K"
+      }
+    },
+    "Hue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "0…360°"
+      }
+    },
+    "Saturation": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "0…100%"
+      }
+    },
+    "Red": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Sources": {
+      "name": "Pre-Mixed Color",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "Generic",
+          "comment": "OFF"
+        },
+        {
+          "dmxRange": [11, 15],
+          "type": "Generic",
```

```suggestion
          "type": "ColorPreset",
```

For the following capabilities, please use `ColorPreset` as well. Also, can you give an RGB approximation for these capabilities? That'd help selecting the right one in lighting programs that show this information next to the capability name.

Use it like so:

```
"type": "ColorPreset",
"colors": ["#ff0000"],
"comment": "Red"
```